### ISSUE TITLE
Clarification of "Partially Completed" implementation

### ISSUE THREAD

**Issue by nuoyanLyu** [Snowflake-Labs/agent-world-model#12]
Hi AWM team,

Thanks for open-sourcing Agent World Model. It is a really impressive piece of work, and I am currently trying to reproduce the RL training setup as faithfully as possible before building on top of it for my own research ideas.

While reading the paper and the public implementation, I noticed one point to be clarified.

In the paper, the code-augmented LLM-as-a-Judge verification design appears to use four outcome classes:
- Completed
- Partially Completed
- Agent Error
- Environment Error

However, in the current `awm/core/verify.py` implementation on `main`, the `sql` mode seems to use the following flow:

- execute verifier code to obtain database verification results
- call `run_llm_judge()`
- ask the LLM judge to choose from:
  - `complete`
  - `incomplete`
  - `server_error`
  - `agent_error`

The prompt also defines `incomplete` as including “partial progress or the database state shows the task is not fully completed”, and the valid classification set is:
```python
{"complete", "incomplete", "server_error", "agent_error", "judge_error"}
```

As a result, I could not find a public-code path where a `Partially Completed` / `partial_success` label is produced or mapped to an intermediate reward such as `0.1`. In the current code, partial progress seems to be folded into `incomplete` / `others`.

Could you clarify the intended behavior?

**Comment by Raibows** [comment]
Hi @nuoyanLyu 

Thank you for your interests in our work! The `incomplete` judge verdict of the code implementation is the `Partially Completed` of the paper.  

`judge_error` is for error or exceptions happened during the process of making a judgment. 

Hope this could help you.
